### PR TITLE
PEP changes and code improvements in agents.ipynb

### PR DESCRIPTION
**Pull request by goswami-rahul**
Also corrected typos in documentation and simplified some code in agents.ipynb.
Minor edit in agents.py

### CODE REVIEW

File: agents.ipynb [hunk 1]
```diff
@@ -134,7 +141,7 @@
    },
    "source": [
     "# PROGRAM - BlindDog #\n",
-    "Now that we have a <b>Park</b> Class, we need to implement a <b>program</b> module for our dog. A program controls how the dog acts upon it's environment. Our program will be very simple, and is shown in the table below.\n",
+    "Now that we have a <b>Park</b> Class, we need to implement a <b>program</b> module for our dog. A program controls how the dog acts in it's environment. Our program will be very simple, and is shown in the table below.\n",
```
Comment (MrDupin) [inline agents.ipynb]: Nice catch on the 'in', but it also should be followed by 'its', with no apostrophe.

Comment (goswami-rahul) [reply agents.ipynb]: Thanks for the review.
Corrected it along with some other typos.

**Comment by goswami-rahul** [comment]
Thanks for the review.
Corrected it along with some other typos.

File: agents.ipynb [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "\n",
     "# AGENT #\n",
     "\n",
-    "An agent, as defined in 2.1 is anything that can perceive its <b>environment</b> through sensors, and act upon that environment through actuators based on its <b>agent program</b>. This can be a dog, robot, or even you. As long as you can perceive the environment and act on it, you are an agent. This notebook will explain how to implement a simple agent, create an environment, and create a program that helps the agent act on the environment based on its percepts.\n",
+    "An agent, as defined in 2.1 is anything that can perceive its <b>environment</b> through sensors, and act upon that environment through actuators based on its <b>agent program</b>. This can be a dog, robot, or even you. As long as you can perceive the environment and act on it, you are an agent. This notebook will explain how to implement a simple agent, create an environment, and create a program that helps the agent to act on the environment based on its percepts.\n",
```
Comment (MrDupin) [inline agents.ipynb]: The 'to' you added is not needed. I think it looks better without it.

File: agents.ipynb [hunk 3]
```diff
@@ -46,7 +46,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "What we have just done is create a dog who can only feel what's in his location (since he's blind), and can eat or drink. Let's see if he's alive..."
+    "What we have just done is created a dog who can only feel what's in his location (since he's blind), and can eat or drink. Let's see if he's alive..."
```
Comment (MrDupin) [inline agents.ipynb]: I think it's 'create', not 'created'.

**Comment by goswami-rahul** [comment]
Updated

**Comment by goswami-rahul** [comment]
@MrDupin is this PR approved or please tell me if any more changes needed?

**Comment by bakerwho** [comment]
@goswami-rahul Peter Norvig merges changes. If you've passed all tests and stuck to style-guides, you should be fine.

**Comment by ad71** [comment]
This PR now contains commits that are already merged. This might lead to merge conflicts later. I'd suggest you close this PR, update your working directory, create a new branch and add just the changes that you originally wanted to make and open a new pull request. Always make a new branch before submitting a pull request or there will be duplicated changes.

**Comment by goswami-rahul** [comment]
This PR is messed up. I think it is because of commits in the master branch. Closing this PR.

**Pull request closed without merging**